tests/templates/kuttl/kerberos-hdfs/70-install-access-hive.yaml.j2

```diff
@@ -0,0 +1,177 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      kubectl apply -n $NAMESPACE -f - <<EOF
+      ---
+      apiVersion: batch/v1
+      kind: Job
+      metadata:
+        name: access-hive
+      spec:
+        template:
+          spec:
+            containers:
+              - name: access-hive
+                image: docker.stackable.tech/stackable/testing-tools:0.2.0-stackable0.0.0-dev
+                imagePullPolicy: IfNotPresent
+                env:
+                  - name: KRB5_CONFIG
+                    value: /stackable/kerberos/krb5.conf
+                  - name: HIVE_METASTORE_HADOOP_OPTS
+                    value: -Djava.security.krb5.conf=/stackable/kerberos/krb5.conf
+                  - name: NAMESPACE
+                    value: $NAMESPACE
+                command:
+                  - /bin/bash
+                  - /tmp/scripts/metastore.sh
+                volumeMounts:
+                  - name: scripts
+                    mountPath: /tmp/scripts/metastore.sh
+                    subPath: metastore.sh
+                  - name: scripts
+                    mountPath: /tmp/scripts/metastore.py
+                    subPath: metastore.py
```

Comment:
Seems like a duplicate
```suggestion
```

Reply:
they're two different files (*.sh, *.py): i'll add this back in

Reply:
2a0782f